compute/instances/api-cli/creating-backups.mdx

```diff
@@ -22,48 +22,97 @@ The Backup feature is used to back up your Instance data. It creates an image of
 - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
 
 
-A backup is created using the command:
-```
-POST https://api.scaleway.com/instance/v1/zones/<region>/servers/<uid>/action
-{"action":"backup"}
-```
+<Tabs id="createbackup">
 
-Replace `<region>` with the geographical region of the Instance. This value can be `fr-par-1` for Instances located in Paris, for example. Replace `<uid>` with the unique identifier of the Instance.
+  <TabsTab label="CLI">
+    ## Creating a backup using the CLI
```

Comment:
Did not think about it, but putting H2 anchors inside tabs breaks the "jump to" menu (anchors that are not visible, i.e. in the active tab cannot be accessed from the "jump to" menu.

An approach could be:

```
## Creating a backup
<Tabs>
<CLI Tab />
<API Tab />
</Tabs>
## Deleting a backup
<Tabs>
<CLI Tab />
<API Tab />
</Tabs>
```